src/main/java/wraith/fwaystones/registry/WaystonesModelProviderRegistry.java

```diff
@@ -20,11 +22,9 @@ public static void register() {
                     return 0f;
                 }
                 if (stack.getItem() instanceof WaystoneScrollItem) {
-                    NbtCompound tag = stack.getNbt();
-                    return tag == null || !tag.contains(FabricWaystones.MOD_ID) || tag.getList(FabricWaystones.MOD_ID, NbtElement.STRING_TYPE).isEmpty() ? 0 : 1;
+                    return 1f;
```

Comment:
Won't this always default to 1 texture, instead of 2 textures per state (1 for empty scroll, 1 for bound scroll)?